Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -922,6 +922,11 @@ - (void)sectionSchema:(WMFExploreSectionSchema*)schema didRemoveSection:(WMFExpl
 - (nullable UIViewController*)previewingContext:(id<UIViewControllerPreviewing>)previewingContext
                       viewControllerForLocation:(CGPoint)location {
     NSIndexPath* previewIndexPath                     = [self.collectionView indexPathForItemAtPoint:location];
```

Comment:
Before the collection view refactor, I think the table view version of "indexPathForItemAtPoint" would actually return an indexPath if you tried to peek a header. Were our headers actually just regular cells then?